SECURITY.md

```diff
@@ -0,0 +1,45 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `Pure Finance`
+project.
+
+- [Reporting a Bug](#reporting-a-bug)
+- [Disclosure Policy](#disclosure-policy)
+- [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+Thank you for improving the security of `Pure Finance`. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the lead maintainer at info@pure.finance by following the [issue template](#).
+
+The lead maintainer will acknowledge your email within 48 hours, and will send a
+more detailed response within 48 hours indicating the next steps in handling
+your report. After the initial reply to your report, the security team will
+endeavor to keep you informed of the progress towards a fix and full
+announcement, and may ask for additional information or guidance.
+
+Report security bugs in third-party modules to the person or team maintaining
+the module.
+
+## Pure Finance bug bounty program
+
+The project engages in an official bug bounty program for security researchers and responsible public disclosures. See https:// for further details.
```

Comment:
Is there a link?

Reply:
Not a direct article but most bounties details are posted on discord channel #dev-general   https://discord.com/invite/Pm8UDaazsK